apps/configs/flameshot.ini

```diff
@@ -6,10 +6,10 @@ disabledTrayIcon=true
 drawColor=#ff0000
 drawFontSize=8
 drawThickness=1
-ignoreUpdateToVersion=0.10.2
-savePath=/home/icedborn/Pictures
+ignoreUpdateToVersion=99.99.99
```

Comment:
This prevents the update notification from showing up (since package is updated via pacman).